### PR TITLE
Add pyproject.toml for Custom Node Registry

### PR DESCRIPTION
**Pull request by haohaocreates**
We are working with dr.lt.data and comfyanon to build a global registry for custom nodes (similar to PyPI). Eventually, the registry will be used as a backend for the UI-manager. All nodes go through a verification process before being published to users.

The main benefits are that authors can
- publish nodes by version and users can safely update nodes knowing ahead of time if their workflows will break or not
- automate testing against new commits in the comfy repo and existing workflows through our CI/CD dashboard

Action Required:

- [x] Go to the [registry](https://registry.comfy.org). Login and create a publisher id. Add the publisher id into the pyproject.toml file.
- [x] Write a short description.
- [ ] Merge the separate Github Actions PR and run the workflow.

If you want to publish the node manually, [install the cli](https://docs.comfy.org/comfy-cli/getting-started#install-cli) and run `comfy node publish`

Check out our [docs](https://docs.comfy.org/registry/overview#introduction) if you want to know more about the registry. Otherwise, feel free to message me on discord at haohao_81202 or join our [server](https://discord.com/invite/comfyorg) if you have any questions!

### CODE REVIEW

**Comment by haohaocreates** [comment]
@digitaljohn, wondering if there's any way we can support you on this PR!

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[project]
+name = "comfyui-propost"
+description = "A set of custom ComfyUI nodes for performing basic post-processing effects including Film Grain and Vignette. These effects can help to take the edge off AI imagery and make them feel more natural."
+version = "1.0.0"
+license = "LICENSE"
```
Comment (digitaljohn) [inline pyproject.toml:5]:
```suggestion
license = "MIT"
```

**Review by digitaljohn — CHANGES_REQUESTED** [review]

**Comment by haohaocreates** [comment]
lgtm!

**Comment by digitaljohn** [comment]
@haohaocreates just need #22 updated and merged then ill merge this in.

**Comment by haohaocreates** [comment]
hey @digitaljohn , sorry about such a long delay, lmk if anything else I can support on!

**Review by digitaljohn — APPROVED** [review]

**Pull request merged**